### PR TITLE
fix: black texture impostors

### PR DESCRIPTION
**Pull request by pravusjif**
https://play.decentraland.zone/?renderer-branch=chore/polish-avatars-lod_fix-black-tex-impostors&NETWORK=mainnet&position=-27%2C60&MAX_VISIBLE_PEERS=200

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRendererHelpers/AvatarRendererHelpers.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@ namespace DCL
 {
     public static class AvatarRendererHelpers
     {
+        private static readonly byte[] questionMarkPNG = { 137, 80, 78, 71, 13, 10, 26, 10, 0, 0, 0, 13, 73, 72, 68, 82, 0, 0, 0, 8, 0, 0, 0, 8, 8, 2, 0, 0, 0, 75, 109, 41, 220, 0, 0, 0, 65, 73, 68, 65, 84, 8, 29, 85, 142, 81, 10, 0, 48, 8, 66, 107, 236, 254, 87, 110, 106, 35, 172, 143, 74, 243, 65, 89, 85, 129, 202, 100, 239, 146, 115, 184, 183, 11, 109, 33, 29, 126, 114, 141, 75, 213, 65, 44, 131, 70, 24, 97, 46, 50, 34, 72, 25, 39, 181, 9, 251, 205, 14, 10, 78, 123, 43, 35, 17, 17, 228, 109, 164, 219, 0, 0, 0, 0, 73, 69, 78, 68, 174, 66, 96, 130, };
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRendererHelpers/AvatarRendererHelpers.cs:7]: We should remove this array as its on the Utils class as well.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRendererHelpers/AvatarRendererHelpers.cs [hunk 2]
```diff
@@ -4,6 +4,7 @@ namespace DCL
 {
     public static class AvatarRendererHelpers
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRendererHelpers/AvatarRendererHelpers.cs:5]:
```suggestion
    public static class AvatarRendererUtils
```
As discussed the static class with pure methods should have the `Utils` suffix.

**Review by AjimenezDCL — APPROVED** [review]
Besides Brian's concerns, all looks good!

**Review by sandrade-dcl — APPROVED** [review]
Aside from what Brian said, LGTM!

**Comment by Suduck** [comment]
lodfail
Not sure if it is related to this PR but I have noticed some weird behaviour on the link in this PR , as you can see there is items loading and the avatars are appearing and dissapearing. Maybe this is the expected behavior but I wanted to notify just in case!

**Pull request merged**